website/src/_posts/2017-02-13-stateful-processing.md

```diff
@@ -293,8 +293,15 @@ new DoFn<KV<MyKey, MyValue>, KV<Integer, KV<MyKey, MyValue>>>() {
 ```
 
 ```py
-# State and timers are not yet supported in Beam's Python SDK.
-# Watch this space!
+class IndexAssigningStatefulDoFn(DoFn):
+  INDEX_STATE = BagStateSpec('index', VarIntCoder())
```

Comment:
As per the discussion, please update this to use CombiningState. (Possibly elsewhere as well.)